src/index.html

```diff
@@ -1,17 +1,169 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="form-field">
+        <legend>
+          Personal Information
+        </legend>
+
+
+        <label class="form__label">
+          Surname:
+          <input class="field"
+            type="text"
+            name="surname"
+            maxlength="10"
+            autocomplete="off">
```

Comment:
```suggestion
          <input 
            class="field"
            type="text"
            name="surname"
            maxlength="10"
            autocomplete="off"
          >
```

проверь везде стиль кода